spec/registrar/register_crawl_object_from_file_spec.rb

```diff
@@ -57,7 +57,7 @@
       valid_file = "#{@fixtures}register_crawl_files/valid_one_line.txt"
       allow(Was::Registrar::RegisterCrawlObjectFromFile).to receive(:verify_file).and_return(true)
       expect(Was::Registrar::RegisterCrawlObjectFromFile).to receive(:convert_line_to_hash).and_return(nil)
-      expect_any_instance_of(Logger).to receive(:fatal).with("Error in registering WAS:Test\tjob1/warc\tdruid:ab123cd4567\tdruid:mn123pq4567 with #<NoMethodError: undefined method `[]' for nil:NilClass>")
+      expect_any_instance_of(Logger).to receive(:fatal).twice
```

Comment:
Actually, I rather like knowing what the expected messages are ... but not something I'll go to the mat over.